### ISSUE TITLE
Rule expect-expect fails for todo

### ISSUE THREAD

**Issue by aarongoldenthal** [jest-community/eslint-plugin-jest#953]
In v25.2.0 (or somewhere between 25.0.5 and 25.2.0), rule `jest/expect-expect` is now failing for `it.todo`.

For example:

```js
describe('some tests', () => {
    it.todo('should have a test');
});
```

```sh
PS E:\Project>npx eslint **/*.js
E:\Project\tests\something.test.js
  2:5  warning  Test has no assertions  jest/expect-expect

✖ 1 problem (0 errors, 1 warning)
```

This occurs in Windows and v12/14/16 node containers with `eslint` 8.0.0/8.0.1.
